desk/ted/lure-check-link.hoon

```diff
@@ -0,0 +1,33 @@
+/-  spider
+/+  *strandio
+=,  strand=strand:spider
+=,  strand-fail=strand-fail:libstrand:spider
+^-  thread:spider
+=/  m  (strand ,vase)
+|^  ted
+++  ted
+  |=  arg=vase
+  ^-  form:m
+  ;<  our=@p   bind:m  get-our
+  =/  arguments  !<((unit [cord ship cord path]) arg)
+  =/  [baseurl=cord target=ship group=cord =path]  (need arguments)
+  =/  target-tape  (trip (scot %p target))
+  ?~  target-tape  !!
+  ;<  ~  bind:m  (send-request %'GET' (url baseurl target group) ~ ~)
+  ;<  rep=client-response:iris  bind:m
+    take-client-response
+  ?>  ?=(%finished -.rep)
+  =/  result  =(200 status-code.response-header.rep)
+  ;<  ~  bind:m  (poke [our %grouper] grouper-link-checked+!>([result path]))
+  (pure:m !>(~))
+++  url
+  |=  [baseurl=cord target=ship group=cord]
+  ^-  cord
+  ?.  =(baseurl 'https://tlon.network/lure/')
+    (crip "{(trip baseurl)}{(trip (scot %p target))}/{(trip group)}")
```

Comment:
should `baseurl` be checked for an ending slash? Also should we maybe move the target tape check to be first since both cases use the target?

Reply:
The input field for baseurl doesn't let you save it unless it has a trailing slash. You're right about moving the check up.